Client/Frontend/Brave Notifications/BraveNotificationsPresenter.swift

```diff
@@ -44,6 +44,7 @@ class BraveNotificationsPresenter: UIViewController {
   private var notificationsQueue: [BraveNotification] = []
   private var widthAnchor: NSLayoutConstraint?
   private var visibleNotification: BraveNotification?
+  private var currentPresntingVC: UIViewController?
```

Comment:
typo and should be weak

Reply:
fixed in [71bd7a3](https://github.com/brave/brave-ios/pull/5271/commits/71bd7a352226f53b3c70d13019e0ece5cb2c70bb)